style.css

```diff
@@ -397,6 +397,90 @@ button:hover {
   width: 100%;
 }
 
+.work-popup {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgba(0, 0, 0, 0.9);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 0 1rem;
+  opacity: 0;
+  visibility: hidden;
+  transition: 0.5s;
+}
+
+.open {
+  opacity: 1;
+  visibility: visible;
+}
+
+.work-popup-content {
+  position: relative;
```

Comment:
- [ ] Good job so far 👏 Your popup window works fine and the popup window is aligned well. Only a little fall out occurs on some project cards where the popup window buttons somehow fall out of view. Take a look at the pic below 👇 
![image](https://user-images.githubusercontent.com/90222110/195420701-c044f94a-2c9a-40f1-8d25-14397623f5c2.png)

TIP: I think you may consider specific height to the image in viewport units or percentages